static/schemas/1.4.0

```diff
@@ -0,0 +1,4 @@
+file_format: 1.0.0
+schema_url: https://opentelemetry.io/schemas/1.4.0
+versions:
+  1.4.0:
```

Comment:
Note that the content of this file is defined by spec, the source of truth here https://github.com/open-telemetry/opentelemetry-specification/blob/main/schemas/1.4.0